frontend/src/metabase/admin/datamodel/metadata/components/MetadataEditor/MetadataEditor.unit.spec.tsx

```diff
@@ -236,6 +245,18 @@ describe("MetadataEditor", () => {
       ).not.toBeChecked();
     });
 
+    it("should display tables with initial_sync_status='incomplete' as disabled", async () => {
+      await setup({ databases: [SAMPLE_DB_WITH_INITIAL_SYNC_INCOMPLETE] });
+      expect(
+        screen.getByText(SAMPLE_DB_WITH_INITIAL_SYNC_INCOMPLETE.name),
+      ).toBeInTheDocument();
+      expect(await screen.findByText("1 Queryable Table")).toBeInTheDocument();
+      const computedStyles = window.getComputedStyle(
+        screen.getByText(ORDERS_TABLE_INITIAL_SYNC_INCOMPLETE.display_name),
```

Comment:
What about testing that you can't click this table by clicking the table and asserting that the right side isn't rendered e.g. you can query that `Select any table to see its schema and add or edit metadata.` is in the screen
![image](https://github.com/metabase/metabase/assets/1937582/e76f6fe7-0882-4fba-ba15-a7ad5b296c10)


This would match user behavior more than just asserting style on the DOM nodes directly which we should avoid.